src/main/webapp/js/finscholar/scholarshippageview/scholarshippageview.soy

```diff
@@ -0,0 +1,109 @@
+{namespace example.templates.scholarshippageviews}
```

Comment:
Missing license header. As a side note, we should go through all the files and make sure all of them have it.

Reply:
Thanks for the reminder!